tests/performance/density.go

```diff
@@ -85,7 +85,7 @@ var _ = SIGDescribe("Control Plane Performance Density Testing", func() {
 	})
 
 	Describe("Density test", func() {
-		vmCount := 100
+		vmCount := 90
```

Comment:
Why is this value changed?

Reply:
The guest-console-log container is adding a certain, small but not null, Requests value for CPU and Memory.
With 100 VMs we hit the limit on Memory while with 90 we are still able to pass the density test.
I tend to think it's an acceptable performance impact.

Reply:
ACK